### ISSUE TITLE
Make print output to the screen

### ISSUE THREAD

**Issue by aaronfreed** [Aleph-Bet-Marathon/alephbet#17]
Corollary of #15 – print (as opposed to Players.print) should output to the screen, to the command line, and to the log by default.

**Comment by SolraBizna** [comment]
I would want there to still be a way for Lua code to output to the log (or stdout, which is the log) on purpose.

**Comment by aaronfreed** [comment]
Right. My proposed behavior is for it to output to three locations by default:
1. The screen.
2. The command line.
3. The log / stdout.
Stretch goal: Option for timestamped log files with the date and time the app was launched (in ISO 8601 format, of course). On Windows, the app currently overwrites existing log files, which I’ve worked around by using a batch file that timestamps the filename. I’ve come to prefer this approach because it stops log files from being frontloaded with a bunch of stuff that’s no longer directly relevant.

**Comment by SolraBizna** [comment]
The "command line output" is stdout. On Windows and macOS, the engine redirects stdout to a log file.

**Comment by aaronfreed** [comment]
Ah, that makes sense. In any case, I think the ideal behavior would be for it to output both to the screen and to stdout.
